lib/jekyll-seo-tag/json_ld.rb

```diff
@@ -40,7 +40,7 @@ def json_author
         return unless author
         {
           "@type" => "Person",
-          "name"  => author["name"],
+          "name"  => author["name"] || page["author"],
```

Comment:
Is this an *alright* kind of way to handle this? It works by looking at the page and fixes my null case issues. (Just like how the title is grabbed in the drop file).